server_ros2/test_client.py

```diff
@@ -0,0 +1,36 @@
+from time import sleep
+
+import rclpy
+
+from sonosco.common.utils import setup_logging
+
+
+import logging
+
+from roboy_cognition_msgs.msg import RecognizedSpeech
+from sonosco.common.constants import SONOSCO
+
+LOGGER = logging.getLogger(SONOSCO)
```

Comment:
Maybe in these test scripts it's better to just use logging and not logger. And perform logging.basicConfig. In this way we'll get the logging from ros as well if they have any.